src/opnsense/mvc/app/controllers/OPNsense/Monit/Api/SettingsController.php

```diff
@@ -310,6 +310,7 @@ public function notificationAction()
                 Config::getInstance()->save();
                 if ($this->mdlMonit->configDirty()) {
                     $result['status'] = 'ok';
```

Comment:
does it need both ? oO

Reply:
I'd say yes.
It's a feedback that all settings were imported correctly.
On the other hand 'No news are good news'... ￼